### PR TITLE
Dev 40

### PR DESCRIPTION
**Pull request by IvanPadalka**
A lot of work is done
image
image
image

### CODE REVIEW

File: src/main/java/com/dimsirka/animalservice/security/WebAuthenticationProvider.java [hunk 1]
```diff
@@ -36,13 +36,13 @@ public Authentication authenticate(Authentication authentication) throws Authent
...
-            if (passwordEncoder.matches(password, userDetails.getPassword())) {
+//            if (passwordEncoder.matches(password, userDetails.getPassword())) {
                 return new UsernamePasswordAuthenticationToken(
                         userDetails.getUsername(),
                         userDetails.getPassword(),
                         userDetails.getAuthorities()
                 );
-            }
+//            }
         }
```
Comment (MarrMika) [inline src/main/java/com/dimsirka/animalservice/security/WebAuthenticationProvider.java:46]: Return it back

File: src/main/java/com/dimsirka/animalservice/controllers/ContactController.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+@Controller
+public class ContactController {
+
+    @GetMapping({"/contact"})
+    public String getAll() {
+        return "contact/contactUs";
+    }
+
+
```
Comment (MarrMika) [inline src/main/java/com/dimsirka/animalservice/controllers/ContactController.java:15]: Move it to HomeController

File: src/main/resources/templates/animal/formCreateAnimal.ftl [hunk 3]
```diff
@@ -1,63 +1,165 @@
...
+<#--                            <th class="align-middle" scope="row">Ім'я тварини:</th>-->
+<#--                            <td class="td-padding">-->
+<#--                                <input name="name" placeholder=""-->
+<#--                                       type="text"/>-->
+<#--                            </td>-->
+<#--                        </tr>-->
+<#--                        <tr>-->
+<#--                            <th class="align-middle" scope="row">Вік тварини:</th>-->
+<#--                            <td class="td-padding">-->
+<#--                                <input name="age" placeholder="" value=""-->
```
Comment (MarrMika) [inline src/main/resources/templates/animal/formCreateAnimal.ftl:42]: Remove commented code if it is useless

File: src/main/resources/templates/animal/info.ftl [hunk 4]
```diff
@@ -1,88 +1,586 @@
...
+                        <#--                        <div class="column">-->
+                        <#--                            <img class="demo cursor" src="img_nature_wide.jpg" style="width:100%"-->
+                        <#--                                 onclick="currentSlide(1)" alt="Nature and sunrise">-->
+                        <#--                        </div>-->
+                        <#--                        <div class="column">-->
+                        <#--                            <img class="demo cursor" src="img_snow_wide.jpg" style="width:100%"-->
+                        <#--                                 onclick="currentSlide(2)" alt="Snow">-->
+                        <#--                        </div>-->
+                        <#--                        <div class="column">-->
+                        <#--                            <img class="demo cursor" src="img_mountains_wide.jpg" style="width:100%"-->
```
Comment (MarrMika) [inline src/main/resources/templates/animal/info.ftl:130]: Remove commented cod if it is useless

File: src/main/resources/templates/animal/info.ftl [hunk 5]
```diff
@@ -1,88 +1,586 @@
...
+                        <td class="td-padding">${animal.animalStatus}</td>
+                    </tr>
+                    </tbody>
+                </table>
+
+                <#--            <h5>Вік:${animal.age}</h5>-->
+                <#--            <h5>Вид:${animal.animalType}</h5>-->
+                <#--            <h5>Дата додавання:${animal.createdDate}</h5>-->
+                <#--            <h5>Статус:${animal.animalStatus}</h5>-->
+                <div class="action">
```
Comment (MarrMika) [inline src/main/resources/templates/animal/info.ftl:180]: Remove commented cod if it is useless

File: src/main/resources/templates/animal/list.ftl [hunk 6]
```diff
@@ -1,150 +1,342 @@
...
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+</div>
+
+<#--<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">-->
+<#--<script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>-->
```
Comment (MarrMika) [inline src/main/resources/templates/animal/list.ftl:27]: Remove commented code if it is useless

File: src/main/resources/templates/animal/list.ftl [hunk 7]
```diff
@@ -1,150 +1,342 @@
...
+                                                                     style="height:220px; visibility: visible;"
+                                                                     src="${animal.mediaLinks[0]}"
+                                                                >
+                                                            <#else>
+                                                                <img class="card-img-top card-item-img"
+                                                                     style="height:220px; visibility: visible;"
+                                                                     src="https://pixy.org/src/120/1206832.jpg"
+                                                                >
+                                                            </#if>
+                                                            <#--                                                        <p id="labelForImg" style=" background-color: #d38960;">Не влаштовано</p>-->
```
Comment (MarrMika) [inline src/main/resources/templates/animal/list.ftl:137]: Remove commented code if it is useless

File: src/main/resources/templates/animal/list.ftl [hunk 8]
```diff
@@ -1,150 +1,342 @@
...
+                                                            <p id="labelForImg" style=" background-color: #ff0000">
+                                                                Потребує
+                                                                допомоги</p>
+                                                            <#--                                                        <p id="labelForImg" style=" background-color: #6096d3;">Влаштовано</p>-->
```
Comment (MarrMika) [inline src/main/resources/templates/animal/list.ftl:141]: Remove commented code if it is useless

File: src/main/resources/templates/order/create.ftl [hunk 9]
```diff
@@ -1,64 +1,142 @@
...
+<#--                        </tr>-->
+<#--                        <tr>-->
+<#--                            <th class="align-middle" scope="row">Ваше номер телефону:</th>-->
+<#--                            <td class="td-padding">-->
+<#--                                <input name="userPhoneNumber" placeholder="" value=""-->
+<#--                                       type="text" tabindex="3" />-->
+<#--                            </td>-->
+<#--                        </tr>-->
+<#--                        <tr>-->
+<#--                            <th class="align-middle" scope="row">Ваша електронна адреса:</th>-->
```
Comment (MarrMika) [inline src/main/resources/templates/order/create.ftl:60]: Remove commented code if it is useless

**Review by MarrMika — CHANGES_REQUESTED** [review]

**Comment by MarrMika** [comment]
image
delete this code

File: src/main/resources/templates/contact/contactUs.ftl [hunk 10]
```diff
@@ -0,0 +1,103 @@
...
+                                                        aria-invalid="false"
+                                                        placeholder="Subject"/></span></p>
+                                        <p><span class="wpcf7-form-control-wrap message"><textarea
+                                                        name="message" cols="40" rows="10"
+                                                        class="wpcf7-form-control wpcf7-textarea"
+                                                        aria-invalid="false"
+                                                        placeholder="Write Here"></textarea></span></p>
+                                        <p><input type="submit" value="Donate Now"
+                                                  class="wpcf7-form-control wpcf7-submit"/></p>
+                                        <div class="wpcf7-response-output wpcf7-display-none"></div>
```
Comment (MarrMika) [inline src/main/resources/templates/contact/contactUs.ftl:90]: Button name doesn't match to the page logic

File: src/main/resources/templates/header.ftl [hunk 11]
```diff
@@ -64,57 +69,58 @@
...
-                                        <li class="nav-item dropdown">
-                                            <a class="nav-link dropdown-toggle" style="color: white; margin: 0 20px;"
+                                        <li class="nav-menu-item-163">
+                                            <a class="menu-item menu-item-type-post_type menu-item-object-page  narrow" style="color: white; margin: 0 20px;"
                                                href="#" id="navbarDropdown" role="button" data-toggle="dropdown"
                                                aria-haspopup="true" aria-expanded="false">
-                                                Замовлення
+                                                Адмінка
                                             </a>
+
```
Comment (MarrMika) [inline src/main/resources/templates/header.ftl:110]: Why adminka?

**Comment by MarrMika** [comment]
image
Align in the center!

**Comment by MarrMika** [comment]
Fix status labels according to their statuses: free, treatment, archived

File: src/main/resources/templates/contact/contactUs.ftl [hunk 12]
```diff
@@ -0,0 +1,103 @@
...
+                                         data-map-height='378'></div>
+                                    <div class='google_map_shortcode_overlay'></div>
+                                </div>
+                                <div class="vc_empty_space" style="height: 37px"><span
+                                            class="vc_empty_space_inner"><span class="empty_space_image"></span></span></div>
+                                <div class="wpb_text_column wpb_content_element ">
+                                    <div class="wpb_wrapper">
+                                        <p>World’s largest no-kill rescue and adoption organization with
+                                            hundreds of dogs, cats, puppies and kittens available</p>
+                                    </div>
```
Comment (MarrMika) [inline src/main/resources/templates/contact/contactUs.ftl:44]: Please, delete this text

File: src/main/resources/templates/contact/contactUs.ftl (hunk 10 above)
Comment (MarrMika) [inline src/main/resources/templates/contact/contactUs.ftl:90]: Align the content in the center

File: src/main/java/com/dimsirka/animalservice/security/WebAuthenticationProvider.java (hunk 1 above)
Comment (IvanPadalka) [reply src/main/java/com/dimsirka/animalservice/security/WebAuthenticationProvider.java:46]: Fixed

File: src/main/java/com/dimsirka/animalservice/controllers/ContactController.java (hunk 2 above)
Comment (IvanPadalka) [reply src/main/java/com/dimsirka/animalservice/controllers/ContactController.java:15]: image
Fixed

File: src/main/resources/templates/animal/formCreateAnimal.ftl (hunk 3 above)
Comment (IvanPadalka) [reply src/main/resources/templates/animal/formCreateAnimal.ftl:42]: Deleted

File: src/main/resources/templates/animal/info.ftl (hunk 4 above)
Comment (IvanPadalka) [reply src/main/resources/templates/animal/info.ftl]: removed

File: src/main/resources/templates/animal/list.ftl (hunk 6 above)
Comment (IvanPadalka) [reply src/main/resources/templates/animal/list.ftl:27]: deleted

File: src/main/resources/templates/animal/list.ftl (hunk 7 above)
Comment (IvanPadalka) [reply src/main/resources/templates/animal/list.ftl:137]: маю реалізувати в цьому пулреквесті тому ще не забираю

File: src/main/resources/templates/animal/list.ftl (hunk 8 above)
Comment (IvanPadalka) [reply src/main/resources/templates/animal/list.ftl:141]: аналогічно

File: src/main/resources/templates/order/create.ftl (hunk 9 above)
Comment (IvanPadalka) [reply src/main/resources/templates/order/create.ftl:60]: Deleted

File: src/main/resources/templates/header.ftl (hunk 11 above)
Comment (IvanPadalka) [reply src/main/resources/templates/header.ftl]: Renamed

File: src/main/resources/templates/contact/contactUs.ftl (hunk 12 above)
Comment (IvanPadalka) [reply src/main/resources/templates/contact/contactUs.ftl:48]: Deleted

File: src/main/resources/templates/contact/contactUs.ftl (hunk 10 above)
Comment (IvanPadalka) [reply src/main/resources/templates/contact/contactUs.ftl]: aligned

File: src/main/resources/templates/animal/info.ftl (hunk 5 above)
Comment (IvanPadalka) [reply src/main/resources/templates/animal/info.ftl]: Deleted

File: src/main/resources/templates/animal/list.ftl (hunk 8 above)
Comment (IvanPadalka) [reply src/main/resources/templates/animal/list.ftl]: Resolved

File: src/main/resources/templates/animal/list.ftl (hunk 7 above)
Comment (IvanPadalka) [reply src/main/resources/templates/animal/list.ftl]: Resolved

File: src/main/resources/templates/contact/contactUs.ftl (hunk 10 above)
Comment (IvanPadalka) [reply src/main/resources/templates/contact/contactUs.ftl]: Fixed

File: src/main/java/com/dimsirka/animalservice/controllers/HomeController.java [hunk 13]
```diff
@@ -5,12 +5,17 @@
 import org.springframework.web.bind.annotation.*;
 
 @Controller
-public class AboutController {
+public class HomeController {
```
Comment (MarrMika) [inline src/main/java/com/dimsirka/animalservice/controllers/HomeController.java:8]: 👍

**Review by MarrMika — APPROVED** [review]
Great work!!!

**Pull request merged**